src/Compilers/Core/Portable/SignatureComparer.cs

```diff
@@ -76,10 +77,19 @@ public bool MatchPropertySignature(PropertySymbol property, ImmutableArray<byte>
         /// Returns true if signature matches signature of the method.
         /// Signature should be in format described in MemberDescriptor.
         /// </summary>
-        public bool MatchMethodSignature(MethodSymbol method, ImmutableArray<byte> signature)
+        public bool MatchMethodSignature(MethodSymbol method, ImmutableArray<byte> signature, int arity)
```

Comment:
>, int arity [](start = 92, length = 11)

It looks like we don't need this parameter, the arity should be available from the ```method```